Stride.BepuPhysics.Navigation/Processors/RecastMeshProcessor.cs

```diff
@@ -73,16 +73,16 @@ public Task RebuildNavMesh()
         // Fetch mesh data from the scene - this may be too slow
         // There are a couple of avenues we could go down into to fix this but none of them are easy
         // Something we'll have to investigate later.
-        var points = new List<Vector3>();
+        var points = new List<VertexPosition3>();
         var indices = new List<int>();
         foreach (var shape in _colliderProcessor.BodyShapes)
         {
             // Copy vertices
             int vBase = points.Count;
-            points.AddRange(shape.Value.Vertex.Select(e => e.Position));
+            points.AddRange(shape.Value.Vertices);
 
             // Copy indices with offset applied
-            indices.Capacity += shape.Value.Indices.Count();
+            indices.Capacity += shape.Value.Indices.Length;
```

Comment:
Who keeps writing `Count()` instead of `Length` :P

Reply:
Ho, didn't know that it was slower. Will try to avoid using it.